src/app/components/HighlightingContext/WithHighlighting.tsx

```diff
@@ -15,7 +15,7 @@ export const WithHighlighting: FC<{ children: ReactNode; address: string }> = ({
         display: 'inline-flex',
         alignItems: 'center',
         verticalAlign: 'middle',
-        padding: '4px 4px 2px 4px',
+        padding: '2px 4px 2px 4px',
```

Comment:
This is where the asymmetry in highlight padding originally came from. It was a bad workaround for other issues, where are now handled properly, so the workaround is gone, and all should be fine.